app/(tabs)/profile.tsx

```
@@ -1,29 +1,28 @@
-import { useUser, useAuth } from '@clerk/clerk-expo';
+import { View, Text, Image, Button, TouchableOpacity, Linking } from 'react-native';
+import { useUser, useAuth, useSession } from '@clerk/clerk-expo';
 import * as ImagePicker from 'expo-image-picker';
-// ^ Make sure this file exists, or remove if unneeded
 import { useRouter } from 'expo-router';
 import { ScreenWrapper } from '../../components/layouts/ScreenWrapper';
 import { ProfileStyles } from '../../styles/Theme';
+import Toast from 'react-native-toast-message';
 import { createClient } from '@supabase/supabase-js';
 
 const supabaseUrl = process.env.EXPO_PUBLIC_SUPABASE_URL!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)